backend/models.py

```
@@ -298,6 +320,44 @@ class UserHint(db.Model):
+        ]:
+            raise ValueError(
+                "Associated module must be of type 'CHALLENGE' or 'BONUS_CHALLENGE'"
+            )
+        return module_id
+
+
+class UserTestCase(db.Model):
+    __tablename__ = "user_test_cases"
+    user_id = db.Column(db.Integer, db.ForeignKey("users.id"), primary_key=True)
+    test_case_id = db.Column(
+        db.Integer, db.ForeignKey("test_cases.id"), primary_key=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)